.readthedocs.yaml

```diff
@@ -0,0 +1,25 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+build:
+  os: ubuntu-22.04
```

Comment:
Is there an `ubuntu-latest` option like in GHA?

Reply:
This (and 20) seemed to be the only two options.